src/quanteda/plot_missing_vals.py

```diff
@@ -0,0 +1,24 @@
+import altair as alt
+import pandas as pd
+
+def plot_missing_vals():
+    """
+    Plot tick chart to display missing values for all numeric features in the dataset.
+
+    Parameters
+    ----------
+    - df : DataFrame
+        the dataset
+
+    Returns
+    -------
+    object
```

Comment:
I looked at altair documentation and this was what they had as the return
```suggestion
    self: object
```